src/StoreKeeper/WooCommerce/B2C/Models/TaskModel.php

```diff
@@ -42,6 +43,7 @@ public static function createTable(): bool
         `type_group` TEXT COLLATE utf8mb4_unicode_ci NOT NULL,
         `storekeeper_id` INT(10) COLLATE utf8mb4_unicode_ci NOT NULL,
         `meta_data` LONGTEXT COLLATE utf8mb4_unicode_ci NOT NULL,
+        `execution_duration` TEXT COLLATE utf8mb4_unicode_ci,
```

Comment:
This won't apply on upgrade of the plugin

Reply:
Added table altering and checking of table if outdated on plugin update.